lib/helper/Appium.js

```diff
@@ -1397,6 +1397,15 @@ class Appium extends Webdriver {
     return super.grabValueFrom(parseLocator.call(this, locator));
   }
 
+  /**
+   * {{> saveScreenshot }}
```

Comment:
Please remove this section (which is common for all helpers)
```suggestion
```
And add basic doc for `saveScreenshot` method without mentioning of a second parameter